docs/blog/2019-01-01-publish-multiple-gatsby-sites/index.md

```diff
@@ -461,9 +461,9 @@ Hopefully what I’ve said makes more sense with the code in front of us. The `s
 
 ### Now
 
-Now is a tool created by ZEIT that makes the process of deploying Node applications simple. Now has recently released updates to the way they deploy static websites. Fortunately our Gatsby packages build to static sites, so it’s a win for us.
+Now is a tool created by Vercel(formally ZEIT) that makes the process of deploying Node applications simple. Now has recently released updates to the way they deploy static websites. Fortunately our Gatsby packages build to static sites, so it’s a win for us.
```

Comment:
i would drop the `formally zeit`

Now -> it should be new: Vercel on all places in this doc

```suggestion
Vercel is a tool created by Vercel that makes the process of deploying Node applications simple. Vercel has recently released updates to the way they deploy static websites. Fortunately our Gatsby packages build to static sites, so it’s a win for us.
```